universal-login-contracts/contracts/common/IMaster.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.5.0;
+
+
+interface IMaster {
+    function owner       ()                            external view returns (address);
+    function master      ()                            external view returns (address);
+    function masterId    ()                            external pure returns (bytes32);
```

Comment:
I would propose to remove ERC725 v2 related functions from IMaster and default Wallet.